### ISSUE TITLE
Need to audit and limit a11y announcements

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16064]
These are interruptive and should be used very sparingly. 

https://master-docs-flutter-io.firebaseapp.com/flutter/semantics/SemanticsService/announce.html

/cc @yjbanov I believe the date/time pickers still use these?
